notes/style-guide.md

```diff
@@ -206,30 +216,29 @@ This is very much a work-in-progress and is not exhaustive.
 #### Preconditions and postconditions
 
 * Preconditions should only be included in names of results if
-  "important" (mostly judgement call).
+  "important" (mostly judgment call).
 
 * Preconditions of results should be prepended to a description
   of the result by using the symbol `⇒` in names (e.g. `asym⇒antisym`)
 
 * Preconditions and postconditions should be combined using the symbols
   `∨` and `∧` (e.g. `m*n≡0⇒m≡0∨n≡0`)
 
-* Try to avoid the need for bracketing but if necessary use square
+* Try to avoid the need for bracketing, but if necessary use square
   brackets (e.g. `[m∸n]⊓[n∸m]≡0`)
 
 #### Operators and relations
 
 * Operators and relations should be defined using mixfix notation where
-  applicable (e.g. `_+_`, `_<_`)
+  applicable (e.g. `_+_`, `_<_`). In a name containing one or more `_`, each underscore can be interpreted as the place where each argument goes.
 
 * Common properties such as those in rings/orders/equivalences etc.
   have defined abbreviations (e.g. commutativity is shortened to `comm`).
   `Data.Nat.Properties` is a good place to look for examples.
 
-* Properties should be by prefixed by the relevant operator/relation
-  (e.g. commutativity of `_+_` is named `+-comm`)
+* Properties should be prefixed by the relevant operator/relation and separated from its name by a hyphen `-` (e.g. commutativity of `_+_` results in a compositional name `+-comm` composed by `+` which denotes the sum operation over the natural numbers, `-` which is the separator of the different sub-names, and `comm` which denotes the commutative property of sum)
```

Comment:
Can this be condensed slightly? Maybe simply a sentence about the use of `-` as a separator. I think we can give the readers some credit and assume they realise what `+` and `comm` refer to.

Reply:
Yes, of course, it can. But i think in general, that the credit given to the readers is not few.